addons/xbmc.python/addon.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="xbmc.python" version="2.24.0" provider-name="Team-Kodi">
-  <backwards-compatibility abi="2.1.0"/>
+<addon id="xbmc.python" version="2.25.0" provider-name="Team-Kodi">
+  <backwards-compatibility abi="2.25.0"/>
```

Comment:
sometimes I think why I am doing all this work. I was informed the my VideoPlayer changes broke an addon. I invested approximately 3 hours to fix this: "with 255 additions and 265 deletions".

you are telling me that addon devs are pissed-off by changing a single line of their config?